06-building-a-program-start-to-finish/writing-tests/output/spec/InvertedIndexSpec.js

```diff
@@ -0,0 +1,277 @@
+const newIndex = new InvertedIndex();
+
+describe('Tests for the InvertedIndex class', () => {
+  localStorage.indexedDocs = JSON.stringify({});
```

Comment:
Nothing, really. It is a vestigial part of the time when I was using localStorage in my class. I have removed it.